site/docs/operations/docker.md

```diff
@@ -359,18 +359,65 @@ To see the default content of the files run the following command:
 docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
 ```
 
-## Use Built-in Modules
+## Image Variants
 
-The image comes with several officially supported connectors. You can
-find out the module names on the
-[Sources and Sinks](../api/sources-sinks.md#summary) page.
+Since the Jet version 4.4, Hazelcast Jet docker images come in two
+different variants, these are:
 
-To use one or more modules provide a list of comma separated names in
-`JET_MODULES` environment variable when running the `docker run`
+### Regular image: hazelcast-jet:<version>
+
+This image includes all of the extension modules in its classpath. It is
+prepared for everything to work out-of-the-box. If you are unsure about
+what your needs are, you probably want to use this one.
+
+### Slim image: hazelcast-jet:<version>-slim
+
+This image only includes Jet core engine and the minimal packages needed
+to run the Jet. It does not contain any extension modules. It should be
```

Comment:
"the Jet" is not like "the Sex-Machine", sounds weird to put "the" in front of it. I would say that all documentation should be mandatory to check with Grammarly.